app/controllers/SummaryController.scala

```diff
@@ -97,19 +97,21 @@ trait SummaryController extends ValidActiveSession {
                      totalGainAndTax: Option[TotalGainAndTaxOwedModel],
```

Comment:
Please can you remove the method `displayAnnualExemptAmountCheck()` above as it isn't being used anymore